src/neural/cuda/fp16_kernels.cu

```diff
@@ -35,26 +35,25 @@ namespace cudnn_backend {
 //          fp16-specific kernels used by certain layers                   //
 /////////////////////////////////////////////////////////////////////////////
 
-
-
 // SE layer implementation using single fused kernel.
 
 // N blocks.
 // C threads per block.
 // 'HWC' input data processed by thread block.
 // Each thread processes 8x8 elements.
 // K is the no. of outputs of first fully connected layer (same as no. of inputs
-// for second fully connected layer). 
+// for second fully connected layer).
 // The kernel assumes K <= C.
 
-#define readw1(row, col) (w1[(row)*K + (col)])
+#define readw1(row, col) (w1[(row)*se_K + (col)])
```

Comment:
This is a curious change that doesn't quite feel like its just formatting or anything specific to do with the rest of the PR.

Reply:
Okay I see why - its because exactly the same defines are in winograd_helper.h and they would be an incorrect redefinition otherwise.

Maybe just remove these and depend on the winograd_helper ones?